Flora/Flora/View Controllers/UserProfileViewController.m

```diff
@@ -39,8 +47,9 @@ - (void)viewDidLoad {
         [self.profPic setImage:[UIImage systemImageNamed:@"person"]];
     }
     
+    self.username.text = [NSString stringWithFormat:@"%@'s boards", self.user.username];
     
-    self.username.text = self.user.username;
+    [self getBoards];
```

Comment:
As a best practice, it's not good to have methods that perform actions that are named `getSomething`. 

`getBoards` implies, to me, that the method is pure & idempotent. The fact that we are calling `[self getBoards]` and not doing with its return value is a code smell.

I recommend renaming `getBoards` to `updateBoards` because it appears that is what you are doing.